4/node.d.ts

```diff
@@ -1362,7 +1362,10 @@ declare module "dgram" {
     export function createSocket(type: string, callback?: (msg: Buffer, rinfo: RemoteInfo) => void): Socket;
 
     interface Socket extends events.EventEmitter {
-        send(buf: Buffer, offset: number, length: number, port: number, address: string, callback?: (error: Error, bytes: number) => void): void;
+        send(msg: Buffer | string | Array<any>, offset: number, length: number, port: number, address: string, callback?: (error: Error, bytes: number) => void): void;
+        send(msg: Buffer | string | Array<any>, offset: number, port: number, address: string, callback?: (error: Error, bytes: number) => void): void;
```

Comment:
In the docs it says that both `offset` and `length` are optional but should either be put together or skipped together. i.e. this overload and the next one aren't needed.